### PR TITLE
Fix deployment issues

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/deployment/app/cumulus_api.template.yml [hunk 1]
```diff
@@ -64,6 +64,17 @@ Resources:
...
+      Policy:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: "Allow"
+            Principal:
+              AWS:
+                Fn::Sub: "arn:aws:iam::${AWS::AccountId}:root"
+            Action:
+              - "execute-api:Invoke"
+            Resource: "execute-api:/*"
```
Comment (mennovandiermen) [inline packages/deployment/app/cumulus_api.template.yml]: I realize it's a dummy, but is this `Resource` accurate? Also, should we wrap the whole policy in the `{{# ifPrivateApi }}`? Lack of resource policy seems to come up on private RestApis only.

Comment (ifestus) [reply packages/deployment/app/cumulus_api.template.yml]: In regards to the `Resource` comment, should this be something like: `arn:aws:execute-api:${region}:${account-id}:${api-id}/${stage}`? Referencing the (Control Who Can Call an API Gateway API Method with IAM Policies)[https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-invoke-api.html]

Comment (laurenfrederick) [reply packages/deployment/app/cumulus_api.template.yml]: @mennovandiermen yeah i think i should put it in the `if`. What do you think the `Resource` should be? Reading through docs I'd think that would work.

Comment (mennovandiermen) [reply packages/deployment/app/cumulus_api.template.yml]: Jacob's suggestion looks fine to me.

**Pull request merged**